modules/core/src/passes/screen-pass-uniforms.ts

```diff
@@ -1,27 +1,31 @@
 import type {TextureView} from '@luma.gl/core';
-import {ShaderModule} from '@luma.gl/shadertools';
-import {UniformTypes} from '../shaderlib/misc/uniform-types';
+import type {ShaderModule} from '../shaderlib/shader-module';
 
 const uniformBlock = `\
 uniform screenUniforms {
   vec2 texSize;
 } screen;
 `;
 
-type ScreenBindingProps = {
+export type ScreenProps = {
   texSrc: TextureView;
-};
-
-type ScreenUniformProps = {
   texSize: [number, number];
 };
 
-export type ScreenProps = ScreenBindingProps & ScreenUniformProps;
+type RenamedUniforms = {
```

Comment:
Why is this called `RenamedUniforms`? Is this as clean as we can make it?

Reply:
This is just testing the the type system, it's not meant to be merged